airflow/providers/google/cloud/transfers/azure_blob_to_gcs.py

```diff
@@ -122,3 +122,21 @@ def execute(self, context: Context) -> str:
                 self.bucket_name,
             )
         return f"gs://{self.bucket_name}/{self.object_name}"
+
+    def get_openlineage_facets_on_start(self):
+        from openlineage.client.run import Dataset
+        from airflow.providers.openlineage.extractors import OperatorLineage
+        wasb_hook = WasbHook(wasb_conn_id=self.wasb_conn_id)
+        connection = wasb_hook.get_connection(wasb_hook.conn_id)
```

Comment:
1. You are correct, `get_conn` is a better way to get the account name instead of getting it from the config. I have updated and pushed the changes. Thanks.
2. based on the naming convention [doc](https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md#wasbs-azure-blob-storage) we have `service_name` after the `@` which means `account_name`.
Account names are globally unique in Azure. [ref](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-overview#storage-account-name)

> Is account enough to uniquely identify this container?

Yes, because Account names are globally unique.

tagging @Adaverse (Author of AzureBlobStorageToGCSOperator) for context